cmd/main.go

```diff
@@ -28,12 +32,26 @@ var (
 	jsonOutput = kingpin.Flag("json-log", "Output log in JSON format").Default("false").Bool()
 
 	completedPath = kingpin.Flag("completed-path", "Path where a 'completion' file will be dropped").Default("/tmp/vault-creds/completed").String()
+	job           = kingpin.Flag("job", "Whether to run in cronjob mode").Default("false").Bool()
+)
+
+var (
+	namespace = os.Getenv("NAMESPACE")
+	podName   = os.Getenv("POD_NAME")
```

Comment:
What happens if those variables are not set? I think it might panic here: https://github.com/uswitch/vault-creds/pull/6/files#diff-4547dddbeb9bb6f58a5933d55575472cR13

Reply:
you get an error if these are not set:
```
{"level":"error","msg":"error getting pod status: error getting pod: resource name may not be empty","time":"2018-03-15T10:34:39Z"}
```